app/src/js/nav.js

```diff
@@ -0,0 +1,30 @@
+export function createNavigation() {
+    const toggleButton = document.querySelector(".usa-accordion__button");
+    const closeButton = document.querySelector(".usa-nav__close");
+    const menu = document.querySelector(".usa-nav");
+    const menuIcon = document.querySelector(".usa-accordion__button svg use");
+
+  if (toggleButton && closeButton && menu) {
+    toggleButton.addEventListener("click", () => {
+      const isOpen = menu.classList.toggle("is-hidden")
+
+        toggleButton.addEventListener("click", () => {
+            const isOpen = menu.classList.toggle("is-hidden");
+
+            menuIcon.setAttribute(
+                "xlink:href",
+                isOpen ? "/assets/img/sprite.svg#close" : "/assets/img/sprite.svg#menu"
```

Comment:
Based on my last comments above, you will have to prepend "/metrics" to these paths, otherwise the icons will not show up on the production build of the website.